convert/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#![feature(duration_constants)]
```

Comment:
Remove file if unused / will not be needed in the future

Reply:
Which one?